### DISCUSSION TITLE
CV - Mary Samanta

### DISCUSSION THREAD

**Discussion by MairySs** [upatras-hci/iv#43] [Show and tell]
Παραθέτω τα links της δραστηριότητας της 2ης εβδομάδας:

[Cv-site](https://mairyss.github.io/)
[Repository](https://github.com/MairySs/MairySs.github.io)

Σύντομα θα υπάρξουν αλλαγές καθώς δεν είμαι πλήρως ευχαριστημένη από το αποτέλεσμα!

**Comment by MairySs** [comment]
### Update:

Έκανα αρκετές αλλαγές στο site με το βιογραφικό μου και άλλαξα εντελώς το θέμα. 
Χρησιμοποίησα το θέμα [cvless ](https://github.com/piazzai/cvless), ενημέρωσα τις ήδη υπάρχων ενότητες με τα στοιχεία μου και πρόσθεσα καινούργιες. Επίσης, έκανα κάποιες γενικές μορφοποιήσεις στο θέμα αλλά και αλλαγές στο αρχείο `particle.js` (στη min μορφή του), ώστε το τελικό αποτέλεσμα της κίνησης των σωματιδίων να είναι πιο ταιριαστό. 

Όλες οι αλλαγές στα αρχεία έγιναν μέσω του `Visual Studio Code` και τα commits μέσω του `PowerShell` των windows καθώς αυτό χρησιμοποιώ και καθημερινά στη δουλειά μου. 

Κατέγραψα μόνο τη διαδικασία του τελευταίου commit γιατί έκανα αρκετές δοκιμές μέχρι το τελικό αποτέλεσμα.
Η καταγραφή έγινε με το εργαλείο [PowerSession](https://github.com/Watfaq/PowerSession-rs) (μια παραλλαγή του asciinema για να υποστηρίζει windows).
Η συγκεκριμένη εγγραφή φαίνεται παρακάτω: 

[![asciicast](https://asciinema.org/a/lAl5hINoxtoBZQl3UhVn5iu2X.svg)](https://asciinema.org/a/lAl5hINoxtoBZQl3UhVn5iu2X)

**Reply by e-liouta** [reply]
Πανέμορφο! Μπορούμε λες να μετατρέψουμε τα αρχεία .html σε .md και να δουλεύει το ίδιο; και το δικό μου html εχει

**Reply by akimo13** [reply]
Πολύ εντυπωσιακό @MairySs , συγχαρητήρια! 👌

**Reply by MairySs** [reply]
@e-liouta , σίγουρα θα υπάρχει κάποιος τρόπος. Το έψαξα και εγώ λίγο, χωρίς αποτέλεσμα, οπότε θέλει παραπάνω χρόνο.
